Library/Homebrew/cask/lib/hbc/artifact/pkg.rb

```diff
@@ -52,8 +54,19 @@ def run_installer(pkg_description)
         ]
         args << "-verboseR" if Hbc.verbose
         args << "-allowUntrusted" if pkg_install_opts :allow_untrusted
+        if pkg_install_opts :choices
+          choices_file = choices_xml
```

Comment:
As it it now, it isn't actually deleted. You should add an `ensure` block at the end of the method.